pipeline/03_graph_db.py

```
@@ -118,22 +157,57 @@ def extract_entities_and_relations(file_name: str, text: str) -> dict:
+
+            if from_name in entity_names and to_name in entity_names:
+                rel["from"] = from_name
+                rel["to"] = to_name
+                filtered_relations.append(rel)
+
+        return {
+            "entities": filtered_entities,
+            "relations": filtered_relations,
+        }
 
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)